.golangci.yml

```diff
@@ -50,9 +50,8 @@ linters:
 issues:
   exclude-rules:
     - linters:
-      # ignore golint error for a lot of packages for now
-      # This is mainly used to avoid errors regarding ids
+      # Used to avoid errors regarding naming conventions
       # We can remove this section once we decide to fix them
       - golint
-      path: "loadtest|coordinator|logger|cmd|api|websocket|terraform"
+      text: "should be.*ID|CamelCase$|FileJSON`$|ConsoleJSON`$|APIURL`$|calling this Response$"
```

Comment:
What are the backticks for?

Reply:
The error message looks like 

```
logger/logger.go:17:2: struct field `ConsoleJson` should be `ConsoleJSON` (golint)
```

It's not part of the regex. It's part of the error message. I'm including it to anchor the exclusion regex to the end of the sentence.

Reply:
Thanks :+1: